react-client/src/components/AwaitMissionOutcome.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class AwaitMissionOutcome extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+  }
+
+  render() {
+
+    return (
+      <div>
+
+      <Stats />
+
+      <Timer seconds={30}/>
+
+        <h3> Awaiting Mission Outcome </h3>
+        (Shown to all players not on the mission [what of the last to vote?])
```

Comment:
as per discussed, we need that after the first few mission participants finish voting, they are taken to this page while the last person votes